src/components/InRequest/InRequestViewCompact.tsx

```diff
@@ -79,6 +140,32 @@ export const InRequestViewCompact: FunctionComponent<IInRequestViewCompact> = (
           <br />
           <Text id="empNameCVId">{formData.empName}</Text>
         </div>
+        {isAuthSSN &&
+          (formData.empType === EMPTYPES.Civilian ||
+            formData.empType === EMPTYPES.Military) &&
+          !formData.closedOrCancelledDate && (
+            <div>
+              <Label weight="semibold" htmlFor="ssnCVId">
+                SSN: (CUI - PRVCY)
+              </Label>
+              <br />
+              <Text style={{ width: "6em", display: "inline-block" }}>
+                {formattedSSN}
+              </Text>
+              {!showSSN ? (
+                <Eye16Regular
+                  onClick={() => {
+                    if (!additionalInfo.data || additionalInfo.isStale) {
+                      additionalInfo.refetch();
+                    }
+                    setShowSSN(window.setTimeout(hideSSN2, 10000));
```

Comment:
When I talked with Celesa, it sounded like that was a good amount of time.  I think we can just implement like this, and adjust if they find it would be more convenient or necessary to be displayed longer.